src/applications/vaos/new-appointment/components/VAFacilityPage/getEligibilityMessage.js

```diff
@@ -43,13 +46,31 @@ export default function getEligibilityMessage({
     (directReason === ELIGIBILITY_REASONS.noClinics ||
       directReason === ELIGIBILITY_REASONS.noMatchingClinics)
   ) {
-    title = 'We couldn’t find a clinic for this type of care';
+    title = 'You can’t schedule this appointment online';
+    const contact = facilityDetails?.telecom?.find(
+      tele => tele.system === 'phone',
+    )?.value;
+
     content = (
       <>
-        We're sorry. This facility doesn’t have any available clinics that
-        support online scheduling for the type of care you selected. Please call
-        this facility to schedule your appointment or search for another
-        facility.
+        <p>
+          We couldn’t find any open appointment times for online scheduling.
+        </p>
+        <p>You’ll need to call the facility to schedule an appointment.</p>
+        <p>
+          <strong>{facilityDetails.name}</strong>
+          <br />
+          {facilityDetails.address?.city},{' '}
+          <State state={facilityDetails.address?.state} />
```

Comment:
May want to check with Peter if the City and State are needed. The Desired Alert State screenshot didn't have it.

Reply:
Is there someone suggesting we include city and state? If so, I am open to exploring that otherwise I would leave out the city and state to keep it concise and stick to what our content specialist recommended.